src/SourceBuild/content/repo-projects/arcade.proj

```diff
@@ -10,9 +10,9 @@
   </ItemGroup>
 
   <ItemGroup>
-    <BuiltSdkPackageOverride Include="@(ArcadeSdkOverride)" />
-    <BuiltSdkPackageOverride Include="@(ArcadeSharedFrameworkSdkOverride)" />
-    <BuiltSdkPackageOverride Include="@(ArcadeCMakeSdkOverride)" />
+    <BuiltSdkPackageOverride Include="Microsoft.DotNet.Arcade.Sdk" Version="$(OutputPackageVersion)" />
+    <BuiltSdkPackageOverride Include="Microsoft.DotNet.SharedFramework.Sdk" Version="$(OutputPackageVersion)" />
+    <BuiltSdkPackageOverride Include="Microsoft.DotNet.CMake.Sdk" Version="$(OutputPackageVersion)" />
```

Comment:
Where is `OutputPackageVerison` defined?

Reply:
`OutputPackageVersion` is currently defined in the `git-info/arcade.props` file that's generated by the VMR. So this doesn't fully get us off the `git-info` folder, but it at least gets us down to the point that we don't have any cross-project usage.

I plan to remove the dependency on the git-info folder for any and all package versions when I switch to the asset manifest model.